packages/wdio-utils/src/driver/utils.ts

```diff
@@ -201,12 +201,38 @@ export function getCacheDir (options: Pick<Options.WebDriver, 'cacheDir'>, caps:
     return driverOptions.cacheDir || options.cacheDir || os.tmpdir()
 }
 
+export function getMajorVersionFromString(fullVersion:string) {
+    let prefix
+    if (fullVersion) {
+        prefix = fullVersion.match(/^[+-]?([0-9]+)/)
+    }
+    return prefix && prefix.length > 0 ? prefix[0] : ''
+}
+
+export async function checkKnownBuild (build: string) {
+    try {
+        const knownGoodVersions: any = await got('https://googlechromelabs.github.io/chrome-for-testing/known-good-versions.json').json()
```

Comment:
We should use `@puppeteer/browsers` primitives here, the package has interfaces that validates whether a given version is available or not.

Reply:
yes that's it, i've added 'canDownload' instead of the part of code